docs/options/external.md

```diff
@@ -0,0 +1,125 @@
+# external
+
+- **Type:** `string | RegExp | (string | RegExp)[] | ((id: string, parentId: string | undefined, isResolved: boolean) => NullValue<boolean>)`
+- **Optional:** Yes ✅
+
+Specifies which modules should be treated as external and not bundled. External modules will be left as import statements in the output.
```

Comment:
```suggestion
Specifies which modules should be treated as external and not bundled. External modules will be left as import statements in the output instead of bring inlined.
```